api/src/main/java/org/openmrs/module/fhir/api/util/FHIRPatientUtil.java

```diff
@@ -160,4 +169,120 @@ public static Patient generatePatient(org.openmrs.Patient omrsPatient) {
 		FHIRUtils.validate(patient);
 		return patient;
 	}
+	
+	public static org.openmrs.Patient generateOmrsPatient(Patient patient) {
+		org.openmrs.Patient omrsPatient=new org.openmrs.Patient();
+
+		List<IdentifierDt> fhirIdList = patient.getIdentifier();
+		IdentifierDt iddt = (IdentifierDt) fhirIdList.get(0);
+		PatientIdentifier idnt = new PatientIdentifier();
+		idnt.setIdentifier(iddt.getValue());
+		Location loc = Context.getLocationService().getLocationByUuid("8d6c993e-c2cc-11de-8d13-0010c6dffd0f");
+		idnt.setLocation(loc);
```

Comment:
@harsha89 @surangak Eventhough disabled location attrubute in  patient identifier type, we have to set a location in the identifier. But in the UI it is displayed as "not applicable" in the location attribute in the identifier field. 

Also the location we set here should be in the databse as well. otherwise it throws an hibernate exception. So what we can do here?

One possible thing is create an dummy location in the databse and set that location in every create request.